test/unit/rum.test.js

```diff
@@ -49,10 +49,6 @@ describe('the RUM API', function () {
     api.getBrowserTimingHeader().should.equal('<!-- NREUM: (0) -->')
   })
 
-  it('should issue a warning outside a transaction', function () {
-    api.getBrowserTimingHeader().should.equal('<!-- NREUM: (1) -->')
-  })
-
```

Comment:
Do we still have a test for `_graceFail(1)`?

Reply:
Yessir, on line 52:
```
  it('should issue a warning if the transaction was ignored', function () {
    helper.runInTransaction(agent, function (tx) {
      tx.ignore = true
      api.getBrowserTimingHeader().should.equal('<!-- NREUM: (1) -->')
    })
  })
```